### PR TITLE
README.md: Drop SoftLayer brand in favor of IBM Cloud

### PR DESCRIPTION
**Pull request by cclauss**
https://www.ibm.com/cloud/info/softlayer-is-now-ibm-cloud

### CODE REVIEW

**Comment by richardlau** [comment]
The Softlayer logo is also in https://github.com/nodejs/build/blob/master/static-assets/tier-2-providers.png.

**Comment by rvagg** [comment]
@sam-github @richardlau any thoughts on whether we need to pass this up through some chain within IBM to get sign-off on branding or would sign-off by you, as representatives, be enough?

**Comment by richardlau** [comment]
> 
> 
> @sam-github @richardlau any thoughts on whether we need to pass this up through some chain within IBM to get sign-off on branding or would sign-off by you, as representatives, be enough?

@rvagg I've asked internally the same question. I'm definitely not in a position to sign-off on branding on behalf of IBM.

cc @mhdawson

**Comment by sam-github** [comment]
I'm in the same position as @richardlau

**Comment by mhdawson** [comment]
I'll follow up.

File: README.md [hunk 1]
```diff
@@ -236,7 +236,7 @@ are required. After that the configuration will be removed.
 [8]:    https://azure.microsoft.com
 [9]:    https://www.joyent.com/
 [10]:   https://www.ibm.com/
-[11]:   https://www.softlayer.com/
+[11]:   https://www.ibm.com/cloud/info/softlayer-is-now-ibm-cloud
```
Comment (mhdawson) [inline README.md]: Not sure this is the correct link.

**Comment by mhdawson** [comment]
I've not forgotten this, just got the right info (I hope) to get the right approval.  Will update once I've gone through the process.

**Comment by cclauss** [comment]
tap... tap... tap... https://www.ibm.com/cloud/info/softlayer-is-now-ibm-cloud

**Comment by refack** [comment]
Added https://github.com/nodejs/build/pull/1645/commits/53c522b987ebac2d2eac33da17dec7f90eab4a9c with an update to
image
Replacing `softlayer` with `packet`

**Comment by mhdawson** [comment]
@cclauss I've been trying to get the approval but there is a bunch of back and forth. Having trouble getting to closure.  But since the update only removes softlayer I think it's ok to land.

**Comment by rvagg** [comment]
Gee, we're a bit out of date aren't we? Voxer dropped us a long time ago and NodeSource should probably be removed too.

**Comment by refack** [comment]
Added https://github.com/nodejs/build/pull/1645/commits/008a1fefa776795ef0d18d2e41b0944ab0294686
with an update to
image
Removing sofylayer, voxer, and nodesource.

**Pull request merged**